crates/miden-objects/src/lib.rs

```diff
@@ -38,6 +38,7 @@ pub use vm_core::{
 
 pub mod assembly {
     pub use assembly::{
+        ast::{Module, ProcedureName, QualifiedProcedureName},
```

Comment:
If we add `ModuleKind` here, then I think we can get rid of the `assembly` crate imports in `crates/miden-lib/src/account/interface/test.rs` and maybe remove it from the dev-dependencies of miden-lib.